pkg/platform/runtime/setup/setup.go

```diff
@@ -568,3 +574,25 @@ func reusableArtifacts(requestedArtifacts []*headchef_models.V1Artifact, storedA
 	}
 	return keep
 }
+
+func formatSolverError(serr *apimodel.SolverError) error {
```

Comment:
I think we should limit this based on length, not lines. 5 lines can still contain a TON of data (you've seen our solver errors).

Reply:
Yes, that makes sense.  There are some newline characters in the output, but the lines can get very large.  I think I'll trim it at 500 characters which woud be 5 lines on a 100 character wide display.